src/main/resources/templates/fragments/header.html

```diff
@@ -6,13 +6,14 @@
 </head>
 <body>
 <center>
-    <h1>Software Endearing Seminar</h1><br/>
-    <h2>Hotel management site</h2><br/><br/>
     <div id="navcontainer" th:fragment="menu">
+        <h1>Software Endearing Seminar</h1><br/>
```

Comment:
It's "Seminar for Software Engineering" or "Software Engineering Seminar"

Reply:
There should not be any `<h1>` headlines in the header, the header should be a title bar with navigation options and small information, like username.